SDKBuildScripts/mac_unity_SetupTestProjects.sh

```diff
@@ -31,10 +31,10 @@ DoWorkTesting () {
     DeleteUnityCruft
     ForcePushD "Assets"
     Nuke "PlayFabSdk"
-    ln -Fvs PlayFabSdk "$WORKSPACE/sdks/$SdkName/Source/PlayFabSDK"
+    ln -Fvs PlayFabSdk "$WORKSPACE/sdks/$SdkName/ExampleTestProject/Assets/PlayFabSDK"
```

Comment:
Avoid dynamic links.
They cause workspaces to become un-deletable, and they can cause workspace deletions to hang, and simultaneous jobs to accidentally share files.
Try to stop using the magic repos and script created projects, and just use the example project in repo.
In fact, I delete almost all of this file in my branch:
https://github.com/PlayFab/SDKGenerator/tree/pg/UnityMac